core/src/main/scala/kafka/server/ControllerServer.scala

```diff
@@ -516,9 +512,9 @@ class ControllerServer(
         controller.close()
       if (quorumControllerMetrics != null)
```

Comment:
Could you please apply to `quorumControllerMetrics`, `controller` and other closable variables?